ui/.env.development

```diff
@@ -1,4 +1,4 @@
-PORT=3001
+PORT=3002
```

Comment:
Why do we need to update the env variable PORT?

Reply:
sorry, a fault operation, commit the test file.